### PR TITLE
Add "Opt-Out" to list-view filter.

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Adds a checkbox to advanced filter (accounts, contacts, leads, prospects) allowing the user to search for emails that are opted out. Also fixes a bug where searching for an opted in email will return emails that have been opted out.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

File: modules/Prospects/metadata/searchdefs.php [hunk 1]
```diff
@@ -79,7 +79,11 @@
                 'label' => 'LBL_ASSIGNED_TO',
                 'function' => array('name' => 'get_user_array', 'params' => array(false))
             ),
-
+            'open_only_opt_out' => array(
```
Comment (samus-aran) [inline modules/Prospects/metadata/searchdefs.php]: Why this name?
open_only_opt_out?

**Comment by pgorod** [comment]
This should be an easy PR to merge, it seems just to be forgotten ;-)

**Comment by Mac-Rae** [comment]
Please rebase this pull request onto hotfix-7.10.x as the hotfix branch is now specifically for 7.11.x issues.

Please take a look at our contribution guidelines for more information.

**Comment by edvinkuric** [comment]
too bad this isn't merged yet :(

do you have any updates here? @Mac-Rae  @Dillon-Brown

**Pull request closed without merging**